src/docs/metrics/realized-value/index.md

```diff
@@ -12,6 +12,9 @@ multiplying all coins/tokens by the last price, every coin/token is assigned the
 price at which it was last moved. Assigning age to coin/tokens is done
 according to the [coin-age model](/metrics/details/stack-coin-age-model).
 
+Realized Value metric is also available for different [time bounds](/metrics/details/timebound), which 
+computes realized value for tokens moved at least once no longer than a specific number of days or years ago.
```

Comment:
This is repeating the Timebound section, do we need it?

Reply:
I added it as a clearer description. Should I remove it?